loopchain/peermanager/peer_loader.py

```diff
@@ -13,57 +13,53 @@
 # limitations under the License.
 """PeerListData Loader for PeerManager"""
 
-import logging
 import os
-from typing import TYPE_CHECKING
 
 from loopchain import configure as conf
 from loopchain import utils
 from loopchain.baseservice import ObjectManager, RestMethod
+from loopchain.blockchain.types import Hash32
 from loopchain.channel.channel_property import ChannelProperty
-from loopchain.peermanager import Peer
-
-if TYPE_CHECKING:
-    from loopchain.peermanager import PeerManager
 
 
 class PeerLoader:
     def __init__(self):
         pass
 
     @staticmethod
-    def load(peer_manager: 'PeerManager'):
-        if os.path.exists(conf.CHANNEL_MANAGE_DATA_PATH):
-            PeerLoader._load_peers_from_file(peer_manager)
+    def load():
+        peers = PeerLoader._load_peers_from_db()
+        if peers:
+            utils.logger.info("Reps data loaded from DB")
+            return peers
+        elif os.path.exists(conf.CHANNEL_MANAGE_DATA_PATH):
+            utils.logger.info(f"Try to load reps data from {conf.CHANNEL_MANAGE_DATA_PATH}")
+            return PeerLoader._load_peers_from_file()
         else:
-            PeerLoader._load_peers_from_rest_call(peer_manager)
+            utils.logger.info("Try to load reps data from other reps")
+            return PeerLoader._load_peers_from_rest_call()
+
+    @staticmethod
+    def _load_peers_from_db() -> list:
```

Comment:
It's a static method. I think `_` that in front of the method name should be removed.